lib.php

```diff
@@ -187,7 +187,7 @@ function zoom_update_instance(stdClass $zoom, ?mod_zoom_mod_form $mform = null)
 
     // Update meeting on Zoom.
     try {
-        zoom_webservice()->update_meeting($zoom);
+        zoom_webservice()->update_meeting($zoom, $mform->_cm->id);
```

Comment:
Assumes $mform will not be null. Would it be safe to null coalesce to `$zoom->coursemodule`? The core module mod_assign seems to use that value.

Reply:
Although `$zoom->coursemodule` appears to be user controlled, if it is always available during both update and add, and the permissions are already being checked by Moodle to determine that the user has the permission to update/add that item, but it may be safe and best to use `$zoom->coursemodule`:
```suggestion
        zoom_webservice()->update_meeting($zoom, $zoom->coursemodule);
```